CONTRIBUTING.md

```diff
@@ -47,47 +48,47 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+your changes to the extension.
 
-- Go to the `demo/` directory of this repo.
+> **Note**: It is not an exhaustive showcase of DVC's features. Testers are
+> encouraged to try other DVC repositories -- especially real-world cases!
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Run `yarn setup:venv` from the root of this repo to set up a virtual Python
+  environment for the demo project (in `demo/.env`).
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Open the `demo/` folder of this repo directly in VS Code.
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+- Pull the project data with the extension (either from the **DVC Tracked**
+  panel in the _File Explorer_ or the **DVC panel** for _Source Control_) or by
+  running `dvc pull` from a _Terminal_.
 
-- Once the environment has been setup you should navigate into the demo folder
-  activate the virtual environment and run `dvc pull`.
+- In order to [run experiments] in the demo project, the virtual env should be
+  loaded through the Microsoft [Python extension] (please install it), having
+  this in `.vscode/settings.json`:
 
-- In order to run experiments in our demo project we require the Python virtual
-  environment to be loaded.
+  ```json
```

Comment:
we probably want to remove this after all?

Reply:
Didn't get why, but it's been removed now.

Reply:
sorry, to clarify. It should be detected automatically when you open the `demo/` project directly in a lot of cases. In some cases it will depend on how DVC is installed, etc. I feel we just can't cover properly different environments with this settings and in a lot of cases, if everything works find it's not needed